src/common/async.cpp

```diff
@@ -42,20 +63,23 @@ struct AsyncState {
     }
 
 void Async::loop_thread(SharedPointer<AsyncState> state) {
-    ight_debug("loop thread entered");
+
+    EvThreadSingleton::ensure();
+
+    ight_debug("async: loop thread entered");
     for (;;) {
 
         LOCKED(
-            ight_debug("loop thread locked");
+            ight_debug("async: loop thread locked");
```

Comment:
In a certain sense, that's what this line of code does: `ight_debug()` is now syntactic sugar for DefaultLogger::get()->debug().

Here in async.cpp I am using the default logger because async code is not bound to any specific test (especially there in the thread that runs in the background).